src/main/nodejs/src/app/playlist-detail/playlist-detail.component.ts

```diff
@@ -74,4 +88,30 @@ export class PlaylistDetailComponent implements OnInit {
     });
   }
 
+  openShareModal(shareModalContent) {
+    this.shareUrl = `${document.location.href}?accessId=${this.playlist.accessKey}`;
+    this.modalService.open(shareModalContent, { size: 'lg' });
+  }
+
+  unlockPlaylist(accessIdInputForm: NgForm) {
+    this.playlistsService.unlockPlaylist(this.playlist.id, this.accessId).subscribe((data: any) => {
+      this.playlist.readOnly = false;
+      accessIdInputForm.reset();
+    });
+  }
+
+  copyToClipboard(text) {
```

Comment:
No need to create a temporary element, pass for the `HTMLInputElement` from the real form instead